php/Terminus/Site.php

```diff
@@ -121,9 +121,13 @@ public function availableEnvironments() {
    */
   public function info($key = null) {
     $info = $this->information;
-    if ($key AND property_exists($info, $key)) {
+    if ($key && property_exists($info, $key)) {
       return $info->$key;
     }
+    if ($key && !property_exists($info, $key)) {
+      // Asking for info on a key that is not found.
+      return null;
+    }
```

Comment:
Could this be rewritten without the duplicate if statements either in a one-liner like this or a nested if statement:

```php
if ($key) {
  return property_exists($info, $key) ? $info->$key : null;
}
```

That might declare the intent of a `$key` resulting in an early return, rather than right now it has some duplicated logic around both `$key` and `property_exists()` that could result in confusion during later refactoring.